php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java

```diff
@@ -360,7 +361,7 @@ public CodeCompletionResult complete(CodeCompletionContext completionContext) {
         switch (context) {
             case DEFAULT_PARAMETER_VALUE:
                 final CaseInsensitivePrefix nameKindPrefix = NameKind.caseInsensitivePrefix(request.prefix);
-                autoCompleteKeywords(completionResult, request, Arrays.asList("array")); //NOI18N
+                autoCompleteKeywords(completionResult, request, Arrays.asList("array", "new")); //NOI18N
```

Comment:
Just curious - should not we use `PHP_GLOBAL_CONST_KEYWORDS` here?

Reply:
Yes, you are right. I'll fix it later (maybe with another fix). Thanks!

Reply:
No problem either, it is this way for years :)